styleguide.md

```diff
@@ -6,15 +8,15 @@ The rules given in this style guide are grouped based on their application to do
 *   **Grammar**: grammatical structures that should prevail
 *   **Text**: how to use text components like lists, tables, and graphics
 
-Each rule in this style guide has a title in the form of a concise statement and one or more recommendations to make the rule practical. When necessary, there is a short explanation which gives reasoning why a specific rule is actually needed.
+Each rule in this style guide has a title in the form of a concise statement and one or more recommendations to make the rule practical. When necessary, there is a short explanation that gives reasoning why a specific rule is actually needed.
 
 **Audience**
 
 We write for a global audience with different skill levels. The audience can be expert DBAs as well as non-DBA users. The language of documentation is standard American English.
 
 **Conformance**
 
-This guide is a living document and can change with time. It may not always match existing documents. The general recommendation is to follow this style guide when writing documentation. If in doubt, use the style guide than matching the style of older documents.
+This guide is a living document and can change with time. It may not always match existing documents. The general recommendation is to follow this style guide when writing documentation. If in doubt, use the style guide than match the style of older documents.
```

Comment:
```suggestion
This guide is a living document and can change with time. It may not always match existing documents. The general recommendation is to follow this style guide when writing documentation. If in doubt, use the style guide instead of matching the style of older documents.
```

Reply:
Rewritten